ors-engine/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/builders/RoadAccessRestrictionsGraphStorageBuilder.java

```diff
@@ -166,6 +168,8 @@ public void processWay(ReaderWay way, Coordinate[] coords, Map<Integer, Map<Stri
                 restrictions = isAccessAllowed(way, VAL_BICYCLE) ? 0 : getRestrictionType(way, VAL_BICYCLE);
             if (RoutingProfileType.isPedestrian(profileType))
                 restrictions = isAccessAllowed(way, "foot") ? 0 : getRestrictionType(way, "foot");
+            if (RoutingProfileType.isUNKNOWN(profileType))
+                restrictions = isAccessAllowed(way, "unknown") ? 0 : getRestrictionType(way, "unknown");
```

Comment:
The second argument to `isAccessAllowed` and `getRestrictionType` corresponds to OSM access-type key, as described in the [OSM wiki](https://wiki.openstreetmap.org/wiki/Key:access). In its most general form it specifies `access` for all modes of transport, e.g. `access = no` says that nobody may legally access the road. This can be overridden by additional tags defining mode of transport such as `bike = yes`. In this context "unknown" is not a valid value.

What I envisioned was to have a fallback value which does not depend on a specific mode of transport, i.e. which just checks based on the value of the `access` tag. For this, it should be enough to have
```java
restrictions = getRestrictionType(way, "access");
```